src/Sentry.Unity.Native/NativeScopeObserver.cs

```diff
@@ -1,59 +1,104 @@
+using System;
+using System.Runtime.InteropServices;
 using Sentry.Extensibility;
-using Sentry.Unity.Json;
-using UnityEngine;
 
-namespace Sentry.Unity.Native
+namespace Sentry.Unity
```

Comment:
Having the type on this namespace is intentional?

Reply:
unintentional, fixed now
```suggestion
namespace Sentry.Unity.Native
```